en-US/index.html

```diff
@@ -1,12 +1,12 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Auto redirection</title>
-</head>
-<body>
-    Welcome to english redirection
-</body>
-</html>
\ No newline at end of file
+    <head>
+        <meta charset="UTF-8" />
```

Comment:
No need "/" at the end of the self-closing tags in HTML 5

Reply:
and no need to indent the code

Reply:
@Sukaato well, the `html` beacons are nesting the `head` and `body` tags so it would be logical to indent those tags to identify the parent-children relationship.